lib/job.js

```diff
@@ -127,12 +131,22 @@ class Job extends BaseModel {
         return /^PR-/.test(this.name);
     }
 
+    /**
+     * Get part of a PR job name based on given type
+     * @method parsePRJobName
+     * @param  {String}    type         Type can either be pr or job, e.g. pr => PR-1, job => main
+     * @return {String}                 Substring of a PR job name based on given type
+     */
+    parsePRJobName(type) {
+        return this.name.match(PR_JOB_NAME)[REGEX_CAPTURING_GROUP[type]];
+    }
```

Comment:
Can we split this into two separate functions one for each type, so that we need not specify the type when used?
Also this could throw error if invoked on non PR job name. Ex: "component". We should check and return `null` if it is not a PR job
```suggestion
    get prName() {
        return this.isPR() ? this.name.match(PR_JOB_NAME)[REGEX_CAPTURING_GROUP['pr']] : null;
    }
    
    get prParentJobName() {
        return this.isPR() ? this.name.match(PR_JOB_NAME)[REGEX_CAPTURING_GROUP['job']] : null;
    }
```

Reply:
I don't think it makes sense to split this into two functions when they're almost literally the exact same code. We should keep the generic one and just pass in the type